src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (true) {
```

Comment:
try to come up with better condition than just leaving true
I'll give you a hint, in each case you compare action.type to something, why would you do that if whole idea of switch is to compare condition to case